### PR TITLE
Test cases: Operational Policy feature

### PR DESCRIPTION
**Pull request by Tharanidk**
This Pr contains two test cases as,
01. Common Policies
02. API Specific Policies

### CODE REVIEW

**Review by chanaka3d — APPROVED** [review]

File: tests/cypress/integration/publisher/017-api-policies/00-common-policy.specs.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        cy.logoutFromPublisher();
+
+    });
+
+    after(function () {
+        //Delete User
+        cy.visit(`${Utils.getAppOrigin()}/carbon/user/user-mgt.jsp`);
+        cy.deleteUser(publisher);
+    })
+})
```
Comment (lasanthaS) [inline tests/cypress/integration/publisher/017-api-policies/00-common-policy.specs.js:78]: Keep new line at the end of the file

File: tests/cypress/integration/publisher/017-api-policies/01-api-specific-policy.specs.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+         });
+    });
+
+    after(function () {
+        //Delete User
+        cy.visit(`${Utils.getAppOrigin()}/carbon/user/user-mgt.jsp`);
+        cy.deleteUser(publisher);
+        
+    })
+})
```
Comment (lasanthaS) [inline tests/cypress/integration/publisher/017-api-policies/01-api-specific-policy.specs.js:105]: Keep new line at the end of the file

File: tests/cypress/integration/publisher/017-api-policies/00-common-policy.specs.js (hunk 1 above)
Comment (Tharanidk) [reply tests/cypress/integration/publisher/017-api-policies/00-common-policy.specs.js:78]: Resolved with commit e159e6db870377cf9dc4726b102cf6292273660e

File: tests/cypress/integration/publisher/017-api-policies/01-api-specific-policy.specs.js (hunk 2 above)
Comment (Tharanidk) [reply tests/cypress/integration/publisher/017-api-policies/01-api-specific-policy.specs.js:105]: Resolved with commit e159e6db870377cf9dc4726b102cf6292273660e

**Pull request merged**